timeseries/setup.py

```diff
@@ -30,7 +30,7 @@
     "statsmodels~=0.13.0",
     "gluonts~=0.12.0",
     "torch>=1.9,<1.14",
-    "pytorch-lightning>=1.7.4,<1.9.0",
+    "pytorch-lightning>=1.7.4,<1.10.0",
```

Comment:
would recommend the following range:

```python3
"pytorch-lightning>=1.8,<1.10",
```

This way we align exactly with multimodal, unless we want for some reason timeseries exclusively to support 1.7.4, but only if installed without `autogluon.multimodal`. @canerturkmen please advise on what you'd prefer.